### PR TITLE
Address few more issues.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [issue id](issue link)
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: docs/local-development.md [hunk 1]
```diff
@@ -257,6 +259,58 @@ To review the concise and clear overview of the pipeline run, use the tkn cli.
 tkn pr describe bare-build-pipeline-run
 ```
 
+The output from above should look something like this:
+```
+Name:              bare-build-pipeline-run
+Namespace:         default
+Pipeline Ref:      python-build-pipeline
```
Comment (ltagliaferri) [inline docs/local-development.md:266]: Maybe we can condense this here:

The output from the above should look something like this (abbreviated):

```
Name:              bare-build-pipeline-run
Namespace:         default
Pipeline Ref:      python-build-pipeline
...
 ∙ bare-build-pipeline-run-list-dependencies      list-dependencies      6 hours ago   6 seconds    Succeeded
 ∙ bare-build-pipeline-run-install-dependencies   install-dependencies   6 hours ago   13 seconds   Succeeded
 ∙ bare-build-pipeline-run-fetch-from-git         fetch-from-git         6 hours ago   8 seconds    Succeeded
 ```

**Review by ltagliaferri — COMMENTED** [review]
I think I would just abbreviate the output block at the `tkn pr describe bare-build-pipeline-run` command

File: docs/local-development.md [hunk 2]
```diff
@@ -4,9 +4,7 @@ Local development environment for exercising Tekton and Sigstore running on kind
 
 **Note**: We have to use `--allow-insecure-registry` due to this [cosign bug](https://github.com/sigstore/cosign/issues/1405).
 
-## Notes for macOS
-
-You may hit file limits; you can run `sudo launchctl limit maxfiles 65536 200000` to remediate that issue.
+## Notes for MacOS
 
 The airplay receiver uses port 5000, which may need to be disabled. Further details via [Apple's developer forum](https://developer.apple.com/forums/thread/682332). Alternatively, you can manually modify the script and change the [REGISTRY_PORT](https://github.com/vaikas/sigstore-scaffolding/blob/main/hack/setup-mac-kind.sh#L19)
```
Comment (ltagliaferri) [inline docs/local-development.md:9]: In Prereqs can add the following Docker requirements:

5 CPUs

2GB Memory

1 GB Swap

Comment (vaikas) [reply docs/local-development.md:9]: Yeah, I put it here, but I'll reformat it :)
https://github.com/Cray-HPE/sigstore-poc/pull/17/files#diff-53e532051b05508d4e9b669b7dbb53f2d317faf2d91cb965e0bf98cc1a2133deR29

**Review by ltagliaferri — APPROVED** [review]
LGTM!

**Comment by vaikas** [comment]
I noticed the golang failure, but given that:
 * These are documentation only changes
 * I'd rather get the documentation up-to-date
Thanks @strongjz (while I was typing I saw you merged it). I'll debug the 1/4 failing golang test to see what might be going on there.

**Pull request merged**